Garbled Circuit.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Garbled Circuit
+parent: Protocols
+grand_parent: Contents
+has_children: true
+nav_order: 1
```

Comment:
I think that under Protocols there is already a `nav_order : 1` which is Yau (there is also a nav_order 2). 
I suggest to change them to 2,3 or to change this file to be number 3

Reply:
Yep, you're right! Will move to number 3.